GVFS/GVFS.UnitTests/Common/Database/PlaceholdersTableTests.cs

```diff
@@ -0,0 +1,435 @@
+﻿using GVFS.Common.Database;
+using GVFS.Tests.Should;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace GVFS.UnitTests.Common.Database
+{
+    [TestFixture]
+    public class PlaceholdersTableTests
+    {
+        private const string DefaultPath = "test";
+        private const byte PathTypeFile = 0;
+        private const byte PathTypePartialFolder = 1;
+        private const byte PathTypeExpandedFolder = 2;
+        private const byte PathTypePossibleTombstoneFolder = 3;
+        private const string DefaultSha = "SHA1";
+
+        [TestCase]
+        public void ConstructorTest()
+        {
+            Mock<IGVFSConnectionPool> mockConnectionPool = new Mock<IGVFSConnectionPool>(MockBehavior.Strict);
+            PlaceholdersTable placeholders = new PlaceholdersTable(mockConnectionPool.Object);
+            mockConnectionPool.VerifyAll();

```

Comment:
What is this testing? That `new PlaceholdersTable()` doesn't call any methods?

Reply:
Yeah basically that the constructor isn't doing any work that needs to be tested so that if some functionality is added to it we will have a test that will need to be updated.